### PR TITLE
HERE positioning with 3d venues and advanced logging sample.

### PR DESCRIPTION
**Pull request by markoturhere**
New example that shows HERE positioning usage with 3d venues. Also shows sample implementation for advanced positioning logging.

### CODE REVIEW

File: positioning-venues-and-logging/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
```
Comment (konovasha) [inline positioning-venues-and-logging/app/build.gradle]: SDK should be changed to 27

File: positioning-venues-and-logging/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    defaultConfig {
+      // Developers must replace this with their own application's package name, which matches
+      // their HERE application ID.
+        applicationId "com.here.example.venuesandlogging"
+        minSdkVersion 16
+        targetSdkVersion 26
```
Comment (konovasha) [inline positioning-venues-and-logging/app/build.gradle]: SDK should be changed to 27

File: positioning-venues-and-logging/app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+repositories {
+    flatDir {
+        dirs 'libs'
+    }
+}
+
+dependencies {
+    implementation(name: 'HERE-sdk', ext: 'aar')
+    implementation 'com.android.support:appcompat-v7:26.1.0'
```
Comment (konovasha) [inline positioning-venues-and-logging/app/build.gradle]: Change to 27.1.1

File: positioning-venues-and-logging/build.gradle [hunk 4]
```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'
```
Comment (konovasha) [inline positioning-venues-and-logging/build.gradle]: Change to 3.1.1

File: positioning-venues-and-logging/gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#Wed Feb 14 13:37:23 EET 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (konovasha) [inline positioning-venues-and-logging/gradle/wrapper/gradle-wrapper.properties]: Change to 4.7

**Comment by markoturhere** [comment]
Why doesn't this project compile?

- Cannot find dependency HERE-sdk.aar
- Cannot find Android plugin

Someh

**Pull request closed without merging**